support/ci/deploy.sh

```diff
@@ -51,12 +57,12 @@ do
   echo "--> Building $component"
   ${TRAVIS_HAB} studio run HAB_CARGO_TARGET_DIR=/src/target build components/${component}
 
-  HART=$(find ./results -name *${component}*.hart)
+  HART=$(find ./results -name *-${component}-*.hart | tail -1)
```

Comment:
would it be better to source `results/last_build.env` and use `results/$pkg_artifact`?

Reply:
yeah. good idea. I always forget about that file.